dj/typing.py

```diff
@@ -237,7 +233,7 @@ def _validate_type(cls, type_: str):
             inner = test.group("inner")
             if outer not in COMPLEX_TYPES:
                 raise ColumnTypeError(f"{outer} is not a KNOWN complex type.")
-            inners = inner.split(",")
+            inners = inner.split(",") if outer == ColumnType.MAP else inner
```

Comment:
What's this doing? Was there a problem with map in particular? all the complex types take comma separated lists right?

Reply:
`Map` and `Row` are pretty similar right? it doesn't affect `Row`?

Reply:
Good point, it should include `Row` here as well

Reply:
Although I think for `Row` it gets more complex because we'll also need to handle cases where there are maps in rows, like:
```
ROW[STR, MAP[STR, STR]]
```

If it's just doing `inner.split(",")`, it would parse to `["STR", "MAP[STR,", "STR]"]`, which isn't right.